data-access/src/app/domain/contexts/cases/service-ticket/v1/service-ticket-v1.ts

```diff
@@ -275,7 +299,18 @@ export class ServiceTicketV1<props extends ServiceTicketV1Props> extends Aggrega
     }
     this.props.description = new ValueObjects.Description(description).valueOf();
   }
+
+
   
+  setAssignedVendorRef(assignedVendor: VendorUserEntityReference) {
```

Comment:
Delete this, setAssignedVendorRef is implemented in the service ticket domain adapter